src/Akka.Streams.Kafka/Messages/Committable.cs

```diff
@@ -13,28 +14,36 @@ public interface ICommittable
         /// <summary>
         /// Commits an offset that is included in a <see cref="CommittableMessage{K,V}"/> 
         /// </summary>
+        [Obsolete("Do not use directly. Use Committer.Flow or Committer.Sink instead.")]
```

Comment:
We want to discourage users from doing a `SelectAsync` + `await .Commit()` because that means losing all of the data-loss protection that's built into the `CommitCollectStage`. We can still use this API internally but users should really use the `Commiter.Sink` or `Committer.Flow` <- those aren't even the correct names for those stages right now due to some really poor API design decisions but that's what they'll eventually be called